aim/sdk/adapters/lightgbm.py

```diff
@@ -0,0 +1,97 @@
+from typing import Optional
+
+from aim import Run
+from aim.ext.resource import DEFAULT_SYSTEM_TRACKING_INT
+
+try:
+    from lightgbm.callback import CallbackEnv
+except ImportError:
+    raise RuntimeError(
```

Comment:
This one will always warn (even if you don't use LightGBM adapter) doesn't it?

Reply:
@mahnerak error will be raised only if you try to import the adapter. as far as we don't have any imports in `aim` or `aim.sdk` modules, we should be fine.
@devfox-se please confirm

Reply:
no there is no problem with it, we have this technique used in all of our callbacks